### DISCUSSION TITLE
How to set the initial position of the window?

### DISCUSSION THREAD

**Discussion by rchrd2** [pmndrs/leva#353] [Q&A]
How does one set the initial position of the window?

For example, if I wanted it to initialize on the bottom right instead of top right, and still be draggable afterwards?

**Comment by dbismut** [comment]
https://github.com/pmndrs/leva/issues/302#issuecomment-1033904324

**Reply by rchrd2** [reply]
Thanks. But doing this seems to replace the draggable behavior. I want to keep the draggable behavior, but just set the initial position. Is this possible?

**Reply by dbismut** [reply]
The fill option shouldn't disable the drag. If it does then it's a bug. Please file an issue with a correlated sandbox showing the problem.

**Reply by rchrd2** [reply]
I stand corrected. Thanks for your help. It does work like you say.
